packages/transactions/src.ts/index.ts

```diff
@@ -256,7 +256,8 @@ export function serialize(transaction: UnsignedTransaction, signature?: Signatur
         if (transaction.accessList != null) {
             logger.throwArgumentError("untyped transactions do not support accessList; include type: 1", "transaction", transaction);
         }
-        return _serialize(transaction, signature);
+        const { type, ...nonTypedTransaction } = transaction;
```

Comment:
Landing here as I experienced the same issue.
Shouldn't `accessList` be destructured too?
```suggestion
        const { type, accessList, ...nonTypedTransaction } = transaction;
```